demos/http/http_demo_plaintext/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+set( DEMO_NAME "http_demo_plaintext" )
+# Demo target.
+add_executable(${DEMO_NAME})
+
+target_sources(
+    ${DEMO_NAME}
+    PRIVATE
+        "${DEMO_NAME}.c"
+)
+
+target_link_libraries(
+    ${DEMO_NAME}
+    PRIVATE
+        http
+        logging
+)
+
+target_include_directories(
+    logging
```

Comment:
Why specifying the include directories for the `logging` and `http` targets here? Wouldn't these libraries have their include directories specified in `logging,cmake` and `http_parser.cmake` respectively?

Reply:
This is just so that `logging` and `http` can find `config.h`. The alternative is to add it as a source.

Reply:
Okay, we can remove the dependency on `config.h` from `logging.c/.h` files to avoid.  For `http`, it is fine for now to do it this way.